src/context/AppContext.js

```diff
@@ -33,36 +33,53 @@ export function useAppContext() {
 }
 
 export default function AppContextProvider({ children }) {
-  const [storedTodo, setStoredTodo] = useLocalStorage("todo", []);
-  const [todoList, todoDispatch] = useReducer(todoReducer, storedTodo);
+  // const [storedTodo, setStoredTodo] = useLocalStorage("todo", []);
+  const [todoList, todoDispatch] = useReducer(todoReducer, []);
   const [pending, setPending] = useState([]);
   const [paused, setPaused] = useState([]);
   const [completed, setCompleted] = useState([]);
   const [user, setUser] = useState(null);
   const [loadingAuth, setLoadingAuth] = useState(true);
+  const [formType, setFormType] = useState("login");
   const [storedDarkMode, setStoredDarkMode] = useLocalStorage("dark", "light");
   const [darkMode, setDarkMode] = useState(storedDarkMode);
```

Comment:
Try having separate contexts for different type of data like one for todo data and other for theme data